mex/model/entities/bibliographic-resource.json

```diff
@@ -0,0 +1,416 @@
+            "sameAs": [
+                "http://purl.org/dc/terms/license"
+            ]
+        },
+        "pages": {
+            "description": "The range of pages or a single page.",
+            "examples": [
+                "1",
+                "45-67",
+                "45 - 67",
+                "II",
+                "XI",
+                "10i"
+            ],
+            "type": "string"
+        },
+        "publicationYear": {
+            "describing": "The year in which the publication was issued.",
+            "examples": [
+                "2011",
+                "1982",
+                "1810"
+            ],
+            "pattern": "^(?:\\d{4})$",
+            "sameAs": "http://datacite.org/schema/kernel-4/publicationYear",
+            "type": "string"
+        },
+        "publisher": {
+            "description": "An entity responsible for making the publication available ([DCT, 2020-01-20](http://dublincore.org/specifications/dublin-core/dcmi-terms/2020-01-20/)).",
+            "items": {
+                "$ref": "/schema/entities/organization#/identifier"
+            },
+            "sameAs": "http://purl.org/dc/terms/publisher",
+            "type": "array"
+        },
+        "respositoryURL": {
+            "anyOf": [
+                {
+                    "$ref": "/schema/fields/link"
+                },
+                {
+                    "type": "null"
+                }
+            ],
+            "default": null,
+            "description": "The handle of the publication in the repository, where the publication is stored."
+        },
+        "section": {
+            "description": "The name of the chapter of the publication, the book section belongs to.",
+            "examples": [
+                "Kapitel 1",
+                "A Section About Public Health",
+                "Chapter XII: The History of Public Health",
+                "12",
+                "A",
+                "B."
+            ],
+            "type": "string"
+        },
+        "subtitle": {
+            "description": "The subtitle of the publication.",
+            "items": {
+                "$ref": "/schema/fields/text"
+            },
+            "subPropertyOf": [
+                "http://purl.org/dc/terms/title"
+            ],
+            "type": "array"
+        },
+        "title": {
+            "description": "The full title of the publication.",
+            "items": {
+                "$ref": "/schema/fields/text"
+            },
+            "minItems": 1,
+            "sameAs": [
+                "http://purl.org/dc/terms/title"
+            ],
+            "type": "array"
+        },
+        "titleOfBook": {
+            "description": "The title of the book in which the book section is published.",
+            "items": {
+                "$ref": "/schema/fields/text"
+            },
+            "minItems": 1,
```

Comment:
if it's minItems=1, it should go into the `required` list at the bottom

Reply:
right, but I did it wrong in the first place. `titleOfBook and `titleOfSeries` must be optional, so I removed the `minItems` for both. 
the rule should be: if `titleOfBook or `titleOfSeries` are given, then it is required. but I guess that is not possible.